tests/phpunit/api/v3/ContributionPageTest.php

```diff
@@ -1988,36 +1988,11 @@ public function testSubmitContributionPageWithPriceSetQuantity($thousandSeparato
     // Set quantity for our test
     $submitParams['price_' . $priceFieldId] = 110;
 
-    // contribution_page submit requires amount and tax_amount - and that's ok we're not testing that - we're testing at the LineItem level
-    $submitParams['amount'] = $this->formatMoneyInput(180 * 16.95 + 110 * 2.95);
     // This is the correct Tax Amount - use it later to compare to what the CiviCRM Core came up with at the LineItem level
     $submitParams['tax_amount'] = (180 * 16.95 * 0.10 + 110 * 2.95 * 0.10);
 
-    $this->callAPISuccess('contribution_page', 'submit', $submitParams);
-    $contribution = $this->callAPISuccessGetSingle('contribution', [
-      'contribution_page_id' => $this->_ids['contribution_page'],
-    ]);
-
-    // Retrieve the lineItem that belongs to the Printing Rights and check the tax_amount CiviCRM Core calculated for it
-    $lineItem1 = $this->callAPISuccessGetSingle('LineItem', [
-      'contribution_id' => $contribution['id'],
-      'label' => 'Printing Rights',
-    ]);
-
-    // Retrieve the lineItem that belongs to the Another Line Item and check the tax_amount CiviCRM Core calculated for it
-    $lineItem2 = $this->callAPISuccessGetSingle('LineItem', [
-      'contribution_id' => $contribution['id'],
-      'label' => 'Another Line Item',
-    ]);
-
-    $finalContribution = $this->callAPISuccess('Contribution', 'getsingle', ['id' => $contribution['id'], 'return' => ['tax_amount', 'total_amount']]);
-
-    $this->assertEquals($lineItem1['line_total'] + $lineItem2['line_total'], round(180 * 16.95 + 110 * 2.95, 2), 'Line Item Total is incorrect.');
-    $this->assertEquals($lineItem1['line_total'] + $lineItem2['line_total'], $finalContribution['total_amount'], 'Contribution total should match line item totals');
```

Comment:
this is now wrong as total is equal to all 3 line items - but validateChecks this